src/lib/updating-element.ts

```diff
@@ -169,6 +173,9 @@ export interface HasChanged {
   (value: unknown, old: unknown): boolean;
 }
 
+export type PropertyDescriptorFactory = (options: PropertyDeclaration,
```

Comment:
Could this type be just: `(options: PropertyDeclaration) => PropertyDescriptor | undefined`?

The return type should be tightened up or documented here. `void` in the type seems to allow for implementations that don't return anything, but that looks like it'll nuke the descriptor at the call site. Likewise, what exactly does `null` signify, and why `null` over `undefined`?

Reply:
Changed.